src/__tests__/to-have-text-content.js

```diff
@@ -83,7 +83,7 @@ describe('.toHaveTextContent', () => {
     )
   })
 
-  test('when matching with empty string and element with content suggest using toBeEmpty instead', () => {
+  test('when matching with empty string and element with content, suggest using toBeEmptyDOMElement instead', () => {
```

Comment:
I wonder why the `toThrowError` line below did not have to be changed as well. Shouldn't the expected error message change to `/toBeEmptyDOMElement()/`?

I think it's because there's a mistake in that regular expression. Those parenthesis are not being matched but they are interpreted as an empty group. Can you please change that regexp to be `/toBeEmptyDOMElement\(\)/`?

Reply:
Nice catch. It slipped past me.

I just changed it. Thanks!